webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2033,6 +2035,8 @@ var CURRENT_GROUP_DATA={};
                             mckInitializeChannel.checkConnected(true);
                         }
                         _this.stopIdleTimeCounter();
+                        clearInterval(FLASH_PAGE_TITLE);
+                        parent.document.title = CURRENT_PAGE_TITLE;
```

Comment:
@CaptainT33mo will this work in case of add kommunicate inside iframe?

Reply:
@devashishmamgain with `parent.document.title` it will not work for those people who are using our widget inside an iframe. And if I use `window.top` then it'll give `Blocked a frame with origin  from accessing a cross-origin frame.` error.